demo/lib/vars/testInParallel.groovy

```diff
@@ -1,5 +1,5 @@
-def call(parallelism, inclusionsFile, exclusionsFile, results, image, prepare, run) {
-  def splits = splitTests parallelism: parallelism, generateInclusions: true
+def call(parallelism, inclusionsFile, exclusionsFile, results, image, stageName, prepare, run) {
+  def splits = splitTests parallelism: parallelism, generateInclusions: true, stage: stageName
```

Comment:
Does this actually work? I think the actual “stage name” as recorded by `junit` will be e.g. `Testing Hello / split5 / Test Section #6` not simply `Testing Hello`.

Reply:
Somehow it seems to work.